### PR TITLE
Refactor the if/else clause for sorting paged list

### PR DESCRIPTION
**Pull request by nwnt**
First time contributing... I think the call is a bit duplicated.  So I extracted the variable and removed the else clause.

### CODE REVIEW

File: calltree-core/src/main/java/org/wicket/calltree/services/ContactServiceImpl.java [hunk 1]
```diff
@@ -146,10 +146,10 @@ public ContactDto fetchContactByPhoneNumber(String string) {
 
     private List<Contact> sortedPagedList(String orderDirection, String orderValue, Integer page, Integer size) {
 
+        Direction dir = Direction.DESC;
```
Comment (TeamWicket) [inline calltree-core/src/main/java/org/wicket/calltree/services/ContactServiceImpl.java]: I like the refactoring, thanks for your suggestion @Thanonchai :)
Perhaps we can make it even better removing completely the IF statement and introducing a ternary operator, something like

`Direction dir = orderDirection.equalsIgnoreCase(ASC) ? Direction.ASC : Direction.DESC;`

and then return it like you did in line 153. What do you think?

I realised that we don't have a unit test to check the correct behaviour of this method. If you're happy to do it, I can create a new issue for this particular case and assigning it to you for another PR. Let me know!

Comment (nwnt) [reply calltree-core/src/main/java/org/wicket/calltree/services/ContactServiceImpl.java]: Oh right, I somehow forgot about the ternary operator altogether.  Yes that would be even cleaner.

For unit test, I'd be happy to do it if you can wait for me to do some trials and errors over the coming week :).

**Review by AlessandroArosio — APPROVED** [review]
Good job @Thanonchai !
Thank you for your contribution

File: calltree-core/src/main/java/org/wicket/calltree/services/ContactServiceImpl.java (hunk 1 above)
Comment (AlessandroArosio) [reply calltree-core/src/main/java/org/wicket/calltree/services/ContactServiceImpl.java]: Sure thing! I'm going to create an issue tomorrow and assign it to you. If you need help with junit, mockito (or even integration test if you like), please reach out. I'll be happy to help :)

**Comment by TeamWicket** [comment]
@Thanonchai issue #92 created for you

**Pull request merged**